workshop/workshop-logistics.md

```diff
@@ -16,7 +16,16 @@ nav_title: Logistics
 
 Our goals for the workshop include the following:
 
-* You will be introduced to `<workshop content>` through a series of hands-on, interactive lessons.
+* You will be introduced to {%- case site.workshop_content -%}
+{% when "intro-r" %}
+  the R programming language
+{% when "intro-single-cell" %}
+  the R programming language and the fundamentals of scRNA-seq analysis
+{% when "intro-bulk" %}
+  the R programming language and the fundamentals of bulk RNA-seq analysis
+{% when "advanced-single-cell" %}
+  advanced scRNA-seq analyses
+{%- endcase -%} through a series of hands-on, interactive lessons.
 * You are able to ask questions and receive 1:1 assistance as needed during instruction. Because our instruction is almost entirely through hands-on lessons, we want to make sure we can help you through any difficulties or errors you may encounter.
```

Comment:
Separating logic from content. Note that I swapped the `{%-` locations. This puts the `case` statement on a separate line, but lets you leave the indentation you have for the `when` content. The result is 3 lines of text for the bullet point, but markdown smooshes them back as you would hope.
```suggestion
* You will be introduced to
{% case workshop_content %}
{%- when "intro-r" -%}
 the R programming language
{%- when "intro-single-cell" -%}
  the R programming language and the fundamentals of scRNA-seq analysis
{%- when "intro-bulk" -%}
  the R programming language and the fundamentals of bulk RNA-seq analysis
{%- when "advanced-single-cell" -%}
  advanced scRNA-seq analyses
{% endcase %}
through a series of hands-on, interactive lessons.
* You are able to ask questions and receive 1:1 assistance as needed during instruction. Because our instruction is almost entirely through hands-on lessons, we want to make sure we can help you through any difficulties or errors you may encounter.
```

Reply:
As long as we get to use the word "smooshes," I'm for it.